src/include/utils.h

```diff
@@ -375,6 +375,62 @@ pgagroal_backtrace(void);
 
 #endif
 
+/**
+ * Command parsing utils for pgagroal-cli and pgagroal-admin.
+ */
+
+#define COMMAND_UNKNOWN (-1)
+
+struct pgagroal_parsed_command
```

Comment:
I would love to have this struct documented, so that it is clear to everyone what fields mean.
Also this structure should be after the `pgagroal_command`, since it is the result of having `pgagroal_command` being parsed.

Reply:
Agreed.